library/worldgen/biome/src/testmod/java/org/quiltmc/qsl/worldgen/biome/QuiltBiomeTest.java

```diff
@@ -93,7 +93,7 @@ public void onInitialize(ModContainer mod) {
 
 		ConfiguredFeature<?, ?> COMMON_DESERT_WELL = new ConfiguredFeature<>(Feature.DESERT_WELL, DefaultFeatureConfig.INSTANCE);
 		Registry.register(BuiltinRegistries.CONFIGURED_FEATURE, new Identifier(MOD_ID, "quilt_desert_well"), COMMON_DESERT_WELL);
-		Holder<ConfiguredFeature<?, ?>> featureEntry = BuiltinRegistries.CONFIGURED_FEATURE.getOrCreateHolder(BuiltinRegistries.CONFIGURED_FEATURE.getKey(COMMON_DESERT_WELL).orElseThrow());
+		Holder<ConfiguredFeature<?, ?>> featureEntry = BuiltinRegistries.CONFIGURED_FEATURE.getHolderOrThrow(BuiltinRegistries.CONFIGURED_FEATURE.getKey(COMMON_DESERT_WELL).orElseThrow());
```

Comment:
why are we switching to `getHolderOrThrow` now? does `getOrCreateHolder` no longer exist?

Reply:
`getOrCreateHolder` returns `DataResult<Holder<ConfiguredFeature<?, ?>>`, and `getHolderOrThrow` return `Holder<ConfiguredFeature<?, ?>>`. I couldn't find a way to change it to not being a DataResult while still using getOrCreate